apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.ts

```diff
@@ -119,6 +148,86 @@ router.post(
           tid_new,
         });
 
+        const serverJob = await editor.prepareServerJob();
+        try {
+          await editor.executeWithServerJob(serverJob);
+          return res.redirect(req.originalUrl);
+        } catch {
+          return res.redirect(res.locals.urlPrefix + '/edit_error/' + serverJob.jobSequenceId);
+        }
+      }
+    } else if (req.body.__action === 'update_assessment') {
+      const infoAssessmentPath = encodePath(
+        path.join(
+          res.locals.course.path,
+          'courseInstances',
+          res.locals.course_instance.short_name,
+          'assessments',
+          res.locals.assessment.tid,
+          'infoAssessment.json',
+        ),
+      );
+      if (!(await fs.pathExists(infoAssessmentPath))) {
+        throw new error.HttpStatusError(400, 'infoAssessment.json does not exist');
+      }
+      const paths = getPaths(undefined, res.locals);
+
+      const assessmentInfo = JSON.parse(await fs.readFile(infoAssessmentPath, 'utf8'));
+      const assessmentInfoEdit = assessmentInfo;
```

Comment:
Why this line?

Reply:
Removed in [ca2ca5d](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/ca2ca5d0e50c8f76d05e06333bbe2e8d3c49cf1c), now just using `assessmentInfo` to write changes.